ci-operator/step-registry/openshift-extended/chain-upgrade/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
```

Comment:
@liangxia I suggest change it to the following
```yaml
approvers:
  - jianzhangbjz
  - kuiwang02
reviewers:
  - jianzhangbjz
  - kuiwang02
  - xia-zhao-rh
```
frankly it is same to upgrade directory.
I ask it because it is golang step and we need to ensure every change is approved by golang framework guy.
should better keep consistent between upgrade step and chain-upgrade step.

Reply:
Updated